packages/core/src/components/hero/hero.css

```diff
@@ -0,0 +1,252 @@
+  --horizontal-padding-small: var(--spacing-layout-2-xs);
+  --horizontal-padding-medium: var(--spacing-layout-xs);
+  --horizontal-padding-large: var(--spacing-layout-xs);
+  --horizontal-padding-x-large: var(--spacing-layout-xs);
+  --padding-horizontal: var(--horizontal-padding-small);
+  --padding-vertical: var(--spacing-xl);
+  color: var(--color);
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  z-index: 1;
+  background-color: var(--background-color);
+}
+.hds-hero h1 {
+  font-size: var(--fontsize-heading-xl);
+  letter-spacing: -1px;
+  font-weight: normal;
+  line-height: 1em;
+  margin-top: 0;
+}
+.hds-hero p {
+  font-size: var(--fontsize-body-l);
+}
+.hds-hero__container {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: var(--container-width-xl);
+}
+.hds-hero__content {
+  display: flex;
+  padding: var(--padding-vertical) var(--padding-horizontal);
+}
+.hds-hero__content--single-column,
+.hds-hero__content--two-columns {
+  display: flex;
+  flex-direction: column;
+}
+.hds-hero__content--two-columns {
+  align-items: center;
+}
+.hds-hero__content--two-columns__image-container {
+  display: none;
+}
+.hds-hero__card {
+  display: block;
+  max-width: var(--container-width-xl);
+}
+.hds-hero__card--centered-content {
+  width: 100%;
+  text-align: center;
+}
+.hds-hero__image {
+  display: block;
+  width: 100%;
+}
+.hds-hero__image-below-koros {
+  display: block;
+}
+.hds-hero__image-below-koros .hds-hero__image {
+  object-fit: cover;
+  height: 100%;
+  object-position: var(--image-position);
+}
+.hds-hero--with-background__container {
+  position: relative;
+}
+.hds-hero--with-background__background {
+  width: 100%;
+}
+.hds-hero--with-background__background .hds-hero__image {
+  object-fit: cover;
+  height: 100%;
+  object-position: var(--image-position);
+}
+
+.hds-hero--with-background__contentColumns {
+  display: flex;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.hds-hero--with-background__empty-column {
+  display: none;
+}
+
+.hds-hero--background-image .hds-hero__content {
+  padding: 0;
+}
+
+.hds-hero--background-image .hds-hero__card {
+  background-color: var(--background-color);
+  padding: var(--padding-vertical) var(--padding-horizontal);
+}
+.hds-hero--background-image__top-koros {
+  display: block;
+}
+.hds-hero--background-image-bottom-koros {
+  display: none;
+}
+
+.hds-hero--diagonal-koros .hds-hero--with-background__container {
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  flex-direction: column;
+}
+.hds-hero--diagonal-koros .hds-hero__content {
+  display: block;
+  z-index: 2;
+  width: 100%;
+  padding: 0;
+  box-sizing: border-box;
+}
+.hds-hero--diagonal-koros .hds-hero__card {
+  background-color: var(--background-color);
+  padding: var(--padding-vertical) var(--padding-horizontal);
+}
+.hds-hero--diagonal-koros .hds-hero--diagonal-koros__koros-and-background {
+  transform: scaleY(-1);
```

Comment:
Yes, but if I use props, the koros sets its own transform-object to style-attribute, which overrides all transformations set with css classes

Reply:
> The sub components might be more flexible because then the lib users can add additional styles via className etc

Hmm... Is 

` <Hero.Title className="custom-class" heading-level={2}>Title</Hero.Title>`
 
more flexible than
 
` <h2 className="custom-class">Title</h2>`
 

The consensus is more with adding `Hero.Title` and `Hero.Text` than using plain HTML, so I'll add them. 

There are no layouts for other headings than h1, so adding heading-level={2} just adds plain h2 without styling.

Reply:
I remember a case where there was a need to configure the heading-levels, at least for the screen-readers
But that would require that the heading is not an h1, h2, or anything like that 🤔
Maybe I'm overthinking this.

Reply:
Done.